opentelemetry-plugin-nginx.yaml

```diff
@@ -0,0 +1,125 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: opentelemetry-plugin-nginx
+  version: 0_git20241009
+  epoch: 0
+  description: Adds OpenTelemetry distributed tracing support to nginx. This is the otel community plugin for nginx, not the official nginx plugin for otel.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - opentelemetry-cpp
+
+environment:
+  contents:
+    packages:
+      - abseil-cpp-dev
+      - busybox
+      - c-ares-dev
+      - cmake
+      - curl-dev
+      - gcc
+      - glibc-dev
+      - grpc-dev
+      - icu-dev
+      - make
+      - nginx
+      - opentelemetry-cpp
+      - pcre-dev
+      - protobuf-c-dev
+      - protobuf-dev
+      - re2-dev
+      - systemd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/open-telemetry/opentelemetry-cpp-contrib
+      branch: main
+      expected-commit: f6d29426ee9b4d6b476c09ca3cb9bed3cf23906f
+
+  - runs: |
+      cd instrumentation/nginx
+
+      mkdir build
+      cd build
+      cmake ..
```

Comment:
`cmake/build` should cover this

Reply:
This can't use `cmake/install` though, since this cmake pipeline outputs directly at the prefix dir and we need to move this to the nginx modules directory.